com.unity.netcode.gameobjects/Runtime/Serialization/FastBufferWriter.cs

```diff
@@ -1185,6 +1186,272 @@ public struct ForFixedStrings
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public void WriteValue(Ray2D[] value) => WriteUnmanaged(value);
 
+        #region UnityMathematics As Value
```

Comment:
we do not use any `#region` blocks as standard

Reply:
Thanks for pointing that out.  I was aware of this, but am using it currently to separate the additions.
I still have to add all of the XML API documentation to each added method which is why it is still in draft.
_(will make sure to remove them when I am done)_